Dockerfile

```diff
@@ -1,6 +1,9 @@
-FROM alpine:latest as builder
+FROM alpine:3.12 as builder
```

Comment:
Keep in mind to avoid using `latest` images as much as possible 😄 

When Alpine will release a `4.x` version, the `latest` will target the `4.x` version which can lead to major changes (for example using another package manager) and, then, make docker builds fail.